msal-client-credentials-flow/README.md

```diff
@@ -0,0 +1,139 @@
+<!-- Keeping yaml frontmatter commented out for now
+---
+# Metadata required by https://docs.microsoft.com/samples/browse/
+# Metadata properties: https://review.docs.microsoft.com/help/contribute/samples/process/onboarding?branch=main#add-metadata-to-readme
+languages:
+- Node.js
+page_type: sample
+name: "Node.js application that makes a request to the Graph API as itself"
+description: "This sample Node.js application demonstrates how to issue a call to a protected API using the client credentials flow.  A request will be issued to Microsoft Graph using the application's own identity."
+products:
+- azure
+- azure-active-directory
+- ms-graph
+urlFragment: ms-identity-docs-code-api-ccf-nodejs
+---
+-->
+
+# Node.js | web API | access control, protected web API access (Microsoft Graph) | Microsoft identity platform
```

Comment:
"access control?" -- what does that mean in this context?

Reply:
removed 'access control'